decrease announcement padding; refactor

nyiyui opened the pull request:
- use `async` over `Promise`
- use `fetch` over `$.getJSON`
- etc

nyiyui committed:
prettify

nyiyui committed:
refactor and show weather as integers

nyiyui committed:
decrease announcement padding

nyiyui committed:
decrease announcement padding

nyiyui committed:
Merge branch 'nyiyui' of github.com:wlmac/tv into nyiyui

nyiyui commented:
> 

Ah,

nyiyui committed:
use async over Promises

nyiyui commented:
> Technically I didn't put it in the html because the qr code loads with the announcements and if the announcements fail itll just be a floating msg

Changed so it's hidden by default (563c5a3964684aa19198dda05830bce831e2d47c).

nyiyui committed:
hide qr code by default

nyiyui committed:
drive all uses of $.getJSON to extinction

nyiyui committed:
faster fullscreen.js; attribution change

nyiyui committed:
use let over var; use arrow functions over …

nyiyui committed:
bump versions

nyiyui committed:
qr: fix bug with wrong selector

rcshim04 commented on js/information.js:
```diff
@@ -343,47 +343,49 @@ function updateSchedule() {
-      $("#temp").text(`${weather.Temperature.Metric.Value.toFixed(1)}°C`);
+      $("#w-icon").show();
+      $("#temp").text(
+        `${Math.round(weather.Temperature.Metric.Value.toFixed(1))}°C`
```
Remove .toFixed(1), and where are you setting the value for weather?

nyiyui replied:
Should be fixed now. Thanks!

nyiyui committed:
fix blank bar on RHS

the blank bar was caused by changing the padding of `.announcement` but
not changing the width and height (they were hard-coded).

nyiyui committed:
fix bug in passing weather
